APISpecification.yaml

```diff
@@ -594,6 +639,19 @@ components:
         - variation_stats
         - regulation_stats
       additionalProperties: false
+    Attributes:
```

Comment:
`Attributes` -> `Attribute`? You are describing one attribute in this object, aren't you?